lib/shortcode/events-manager.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/**
+ * Shortcode Template File for Events Manager
+ */
+
+/**
+ * Create our shortcode for Events Manager
+ */
+add_action( 'init', 'sandwich_events_manager', 11 );
+function sandwich_events_manager() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
+		return;
+	}
+
+	// Ensure plugin runs only in admin mode. Do not continue if that's not the case.
+	if ( ! is_admin() ) {
+		return;
+	}
+
+	// Dependency check for Events Manager. Do not proceed if not found.
+	if ( ! defined( 'EM_VERSION' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
+		array(
+			'label' => __( 'Events Manager - Events List', 'pbsandwich' ),
+			'listItemImage' => 'dashicons-calendar',
+			'attrs' => array(
+				array(
+					'label' => __( 'Event List caption', 'pbsandwich' ),
+					'attr' => 'content',
+					'type' => 'textarea',
+					'value' => '',
+					'description' => __( 'The Event List can be customized to include customized description. The following variables will be replaced with the actual relevant details:<br />#_EVENTNAME<br />#_EVENTLINK<br />#_LOCATIONLINK<br />#_EVENTDATES<br />#_EVENTTIMES', 'pbsandwich' ),
+				),
+				array(
+					'label' => __( 'Event listing limit', 'pbsandwich' ),
+					'attr' => 'limit',
+					'type' => 'text',
+					'value' => '10',
+					'description' => __( 'Show up to the specified amount of events.', 'pbsandwich' ),
+				),
+			),
+		)
+	);
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
```

Comment:
There are 2 declarations for the shortcode `events_list`. Is the earlier one needed?